### PR TITLE
ADMINAPI-336 | Structure a file layout for the packaged binaries

### PR DESCRIPTION
**Pull request by lchavarriaga**

### CODE REVIEW

File: Docker/dbadmin.Dockerfile
Comment (CSR2017) [inline Docker/dbadmin.Dockerfile:1]: @lchavarriaga 
Please update the files path on COPY statements.
ex: COPY Settings/DB-Admin/pgsql/run-adminapi-migrations.sh /docker-entrypoint-initdb.d/3-run-adminapi-migrations.sh

File: Docker/api-mssql.Dockerfile
Comment (CSR2017) [inline Docker/api-mssql.Dockerfile:1]: @lchavarriaga 
Please update the files path on COPY statements.

File: Docker/api-pgsql.Dockerfile
Comment (CSR2017) [inline Docker/api-pgsql.Dockerfile:1]: @lchavarriaga
Please update the files path on COPY statements.

File: Docker/Compose/pgsql/compose-build-dev.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ services:
       # Important to set the context to the root, rather than setting it to the
       # EdFi.Ods.AdminApi directory, so that the Dockerfile has access to other
       # C# projects.
-      context: ../../../../
+      context: ../../../
       dockerfile: Application/EdFi.Ods.AdminApi/dev.Dockerfile
```
Comment (CSR2017) [inline Docker/Compose/pgsql/compose-build-dev.yml:76]: @lchavarriaga 
The context path and docker file paths are wrong.

File: build.ps1 [hunk 2]
```diff
@@ -443,6 +444,43 @@ function Invoke-PushPackage {
     Invoke-Step { PushPackage }
 }
 
+function CopyTempFilesToDockerContext {
+	$sourceArtifacts = "$solutionRoot/EdFi.Ods.AdminApi/Artifacts/"
+	$destinationArtifacts = "$dockerRoot/"
```
Comment (CSR2017) [inline build.ps1:449]: @lchavarriaga 
The Artifacts folder is under of "EdFi.Ods.AdminApi". So we can read it from there. Copying EdFi.Ods.AdminApi should be enough

File: Docker/dbadmin.Dockerfile [hunk 3]
```diff
@@ -10,9 +10,9 @@ ENV POSTGRES_USER=${POSTGRES_USER}
 ENV POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
 ENV POSTGRES_DB=postgres
 
-COPY Docker/DB-Admin/pgsql/run-adminapi-migrations.sh /docker-entrypoint-initdb.d/3-run-adminapi-migrations.sh
+COPY Settings/DB-Admin/pgsql/run-adminapi-migrations.sh /docker-entrypoint-initdb.d/3-run-adminapi-migrations.sh
 COPY Artifacts/PgSql/Structure/Admin/ /tmp/AdminApiScripts/PgSql
```
Comment (CSR2017) [inline Docker/dbadmin.Dockerfile:14]: @lchavarriaga 
Path needs to be updated to match folder structure on docker context.

File: .github/workflows/api-e2e.yml [hunk 4]
```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Run Admin API
```
Comment (CSR2017) [inline .github/workflows/api-e2e.yml:39]: @lchavarriaga 
CopyToDockerContext, RemoveDockerContextFiles functions should be called from here to copy over the required files before docker compose.

Comment (CSR2017) [reply .github/workflows/api-e2e.yml:47]: @lchavarriaga 
I added the steps on api-e2e.yml to copy those required application folder and NuGet.Config file to Docker/ Application folder.

**Review by CSR2017 — APPROVED** [review]

**Review by jasonh-edfi — APPROVED** [review]
Administrative review and changes here are in line with request to optimize Admin API Docker deployments.

**Pull request merged**